m4_sessions/1811_inning/women_in_tech.md

```diff
@@ -25,6 +32,17 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Intro - introductions of participants.
+Presenter's Favorite Women - we talk about our favorite women in the history of tech in depth (10 minutes)
+Lesson - give students a sheet of 20 women and their achievements to learn. (15 minutes)
+Pom - (5 minutes)
+Game - Split into 2 teams and put the women's names in a bowl. They draw from the bowl and talk about the women and their team guesses which woman they picked. (20 minutes)
+Pom - (5 minutes)
+Discussion - Give students discussion questions (10 minutes)
+Close - (5 minutes)
+Discussion Questions;
+-Describe a time you've seen discrimination against women working in tech.
```

Comment:
What new information did you learn today?  
How can you bring this new information to your cohort and the people around you?
What does this make you think about your future in development?